yeonjae22/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/data/ProductRemoteDataSource.kt

```diff
@@ -12,9 +12,8 @@ import com.example.elevenstreet.ProductXmlPullParserHandler
 import java.net.URL
 
 
-class ProductRemoteDataSource() {
+class ProductRemoteDataSource private constructor(private var elevenStreetApi: ElevenStreetApi? = null) {
```

Comment:
```suggestion
class ProductRemoteDataSource private constructor(private val elevenStreetApi: ElevenStreetApi) {
```
아래에 리뷰가 안달리는데
```
elevenStreetApi =
            RetrofitInstance.getInstance<ElevenStreetApi>("https://openapi.11st.co.kr/openapi/")
```
이걸 제거해주세요. 결국 안에서 생성하고 있습니다.

Reply:
수정했습니다